labs/jack-in/README.md

```diff
@@ -0,0 +1,18 @@
+# Jack-in
+_your experimental terminal to connect to  matrix via sliding sync_
+
+A simple example client, using sliding sync to [jack in](https://matrix.fandom.com/wiki/Jacking_in) to matrix via the sliding-sync-proxy. 
+
+## Use:
+
+You will need a [locally running sliding sync proxy]() for now.
+
+### Get the access token
+
+1. Login with a fresh session on [element.io](https://develop.element.org)
```

Comment:
This will do two things:

1. Create a new device with a server provided device ID and access token
2. Upload fresh E2EE keys that are tied to this device ID.

We warn against this scenario in the [docs](https://matrix-org.github.io/matrix-rust-sdk/matrix_sdk/encryption/index.html#using-an-access-token). Perhaps we should make it more prominent and easier to find.